<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditoriumDeleteForm</class>
 <widget class="QWidget" name="AuditoriumDeleteForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>301</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>791</width>
    <height>301</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Аудитория, удаление</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AuditoriumDeleteForm {

	background-color: #284B63;
}</string>
  </property>
  <widget class="QComboBox" name="bodyBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox#bodyBox {

	border: 5px solid #1A3344;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#text1 {

	background-color: #284B63;
	color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Корпус - этаж - номер</string>
   </property>
  </widget>
  <widget class="QLabel" name="headText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#headText {

	background-color: #FFC300;
	color: #1A3344;
	border: 5px solid #1A3344;
}</string>
   </property>
   <property name="text">
    <string>Удаление аудитории</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#deleteButton {

	background-color: #284B63;
	color: #FFFFFF;
	border: 5px solid #FFFFFF;
}
QPushButton#deleteButton:hover {

	background-color: #FFFFFF;
	color: #1A3344;
	border: 5px solid #1A3344;
}
QPushButton#deleteButton:pressed {

	background-color: #DB1921;
	color: #FFFFFF;
	border: 5px solid #1A3344;
}
QPushButton#deleteButton:disabled {

	background-color: #284B63;
	color: #4B7089;
	border: 5px solid #4B7089;
}</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="logHead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#logHead {

	background-color: #FFC300;
	color: #1A3344;
	border: 5px solid #1A3344;
}</string>
   </property>
   <property name="text">
    <string>Лог ошибок и предупреждений</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="logText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>771</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit#logText {

	border: 5px solid #1A3344;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>381</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>381</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#exitButton {

	background-color: #284B63;
	color: #FFFFFF;
	border: 5px solid #FFFFFF;
}
QPushButton#exitButton:hover {

	background-color: #FFFFFF;
	color: #1A3344;
	border: 5px solid #1A3344;
}
QPushButton#exitButton:pressed {

	background-color: #DB1921;
	color: #FFFFFF;
	border: 5px solid #1A3344;
}
QPushButton#exitButton:disabled {

	background-color: #284B63;
	color: #4B7089;
	border: 5px solid #4B7089;
}</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QComboBox" name="floorBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox#floorBox {

	border: 5px solid #1A3344;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="numberBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox#numberBox {

	border: 5px solid #1A3344;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
